src/sqlfluff/dialects/dialect_ansi_keywords.py

```diff
@@ -685,6 +686,8 @@
 SERVER_NAME
 SESSION
 SESSION_USER
+SET_DEFAULT
+SET_NULL
```

Comment:
Remove these

Reply:
Running `tox -e generate-fixture-yml,py38` results in `RuntimeError: Grammar refers to 'Set_nullKeywordSegment' which was not found in the ansi dialect` then.

Reply:
Read the suggested advice in the first PR